utils/contract_data_fetchers.py

```diff
@@ -36,7 +36,9 @@ def _get_int_view(self, view_name: str, attrs) -> int:
         result = None
         try:
             result = self._query_contract(view_name, attrs)
-            if result.return_data[0] == '':
+            if len(result.return_data) == 0:
```

Comment:
if <statement> or <statement>

Reply:
Don't know how I ended doing it this way. Fixing it.